src/components/general/DatePicker.js

```diff
@@ -3,195 +3,132 @@ import PropTypes from "prop-types";
 import { View } from "react-native";
 import { Input } from "./Input";
 import { PickerInput } from "./PickerInput";
```

Comment:
Import from "./", since `index.js` takes care of it, in case the file name is changed.